src/libs/runtime-utils.js

```diff
@@ -10,6 +10,16 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
+export const styles = {
```

Comment:
Can we have a slightly more descriptive name? I'm imagining a possible future world where we consolidate multiple large shared style objects here, and need to differentiate.